R/connect.R

```diff
@@ -830,12 +830,12 @@ Connect <- R6::R6Class(
     #' @param name The package name to filter by.
     #' @param page_number The page number.
     #' @param page_size Page size, max 500.
-    packages = function(name = NULL, page_number = 1, page_size = 500) {
+    packages = function(name = NULL, page_number = 1) {
```

Comment:
I'm not opposed to having this as an argument if it defaults to something sensible. In case someone really does know what they're doing and wants to fiddle with it that's ok. My complain in prior PRs was that we _force_ folks to deal with this in cases where they don't want to. 

But if you would rather hide it entirely, that's fine.